.github/workflows/CI.yml

```
@@ -42,8 +42,8 @@ jobs:
           arch: ${{ matrix.arch }}
       - uses: julia-actions/cache@v1
         with:
-          cache-name: CI - Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
-          cache-compiled: true
+           cache-name: CI - Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+           cache-compiled: true
```

You added some extra whitespace:
```suggestion
          cache-name: CI - Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
          cache-compiled: true
```